mobile/src/main/java/com/gelakinetic/mtgfam/fragments/TradeFragment.java

```diff
@@ -29,309 +26,255 @@
+     *                           this can be used to generate the LayoutParams of the view.
+     * @param savedInstanceState If non-null, this fragment is being re-constructed from a previous
+     *                           saved state as given here.
      * @return The inflated view
      */
     @Override
-    public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState) {
+    public View onCreateView(
+            LayoutInflater inflater,
+            ViewGroup container,
+            Bundle savedInstanceState) {
 
-        /* Inflate the view, pull out UI elements */
+        /* Inflate the view, pull out the UI elements */
         View myFragmentView = inflater.inflate(R.layout.trader_frag, container, false);
+
         assert myFragmentView != null;
-        mNameEditText = (AutoCompleteTextView) myFragmentView.findViewById(R.id.namesearch);
-        mNumberEditText = (EditText) myFragmentView.findViewById(R.id.numberInput);
-        mCheckboxFoil = (CheckBox) myFragmentView.findViewById(R.id.trader_foil);
-        mTotalPriceRight = (TextView) myFragmentView.findViewById(R.id.priceTextRight);
+        mNameField = (AutoCompleteTextView) myFragmentView.findViewById(R.id.name_search);
+        mNumberOfField = (EditText) myFragmentView.findViewById(R.id.number_input);
+        mCheckboxFoil = (CheckBox) myFragmentView.findViewById(R.id.list_foil);
+
+        /* Set up the autocomplete adapter, and default number */
+        mNameField.setAdapter(
+                new AutocompleteCursorAdapter(this,
+                        new String[]{CardDbAdapter.KEY_NAME},
+                        new int[]{R.id.text1}, mNameField,
+                        false)
+        );
+        mNumberOfField.setText("1");
+
+        /* Initialize the left list and company */
+        mListLeft = new ArrayList<>();
+        mListAdapterLeft = new CardDataAdapter(mListLeft);
+        mListViewLeft = (RecyclerView) myFragmentView.findViewById(R.id.tradeListLeft);
+        mListViewLeft.setAdapter(mListAdapterLeft);
+        mListViewLeft.setLayoutManager(new LinearLayoutManager(getContext()));
+        ItemTouchHelper.SimpleCallback leftCallback =
+                new SelectableItemTouchHelper(mListAdapterLeft, ItemTouchHelper.LEFT);
+        leftItemTouchHelper = new ItemTouchHelper(leftCallback);
+        leftItemTouchHelper.attachToRecyclerView(mListViewLeft);
+
+        /* Initialize the right list and company */
+        mListRight = new ArrayList<>();
+        mListAdapterRight = new CardDataAdapter(mListRight);
+        mListViewRight = (RecyclerView) myFragmentView.findViewById(R.id.tradeListRight);
+        mListViewRight.setAdapter(mListAdapterRight);
+        mListViewRight.setLayoutManager(new LinearLayoutManager(getContext()));
+        ItemTouchHelper.SimpleCallback rightCallback =
+                new SelectableItemTouchHelper(mListAdapterRight, ItemTouchHelper.LEFT);
```

Comment:
Derp. I just saw one LEFT in a sea of RIGHTs and assumed it was a copy/paste bug. We did talk about this before.